spec/package_spec.rb

```diff
@@ -293,6 +287,11 @@ def build_app_source_with_files(files)
     it 'ignores "description"' do
       expect(subject['description']).to be nil
     end
+
+    it 'preserves the rest' do
+      puts subject
```

Comment:
Shall it be removed then? Otherwise it would pollute the test output, wouldn't it?